<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarksWindow</class>
 <widget class="QDialog" name="MarksWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="description_label">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="date_event_label">
         <property name="text">
          <string>Date of event:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weight_label">
         <property name="text">
          <string>Weight:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="date_mark_label">
         <property name="text">
          <string>Date of mark:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="info_label">
         <property name="text">
          <string>Other info:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add / Change mark</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addEventButton">
       <property name="text">
        <string>Add event</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteEventButton">
       <property name="text">
        <string>Delete event</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
